cluster-operator/src/main/java/io/strimzi/operator/cluster/model/PodDisruptionBudgetUtils.java

```diff
@@ -124,43 +91,5 @@ public static PodDisruptionBudget createCustomControllerPodDisruptionBudget(
                 .build();
     }
 
-    /**
-     * Creates a PodDisruptionBudget V1Beta1 for use with custom pod controller (such as StrimziPodSetController). Unlike
-     * built-in controllers (Deployments, StatefulSets, Jobs, DaemonSets, ...), custom pod controllers can use only PDBs
-     * with minAvailable in absolute numbers (i.e. no percentages).
-     * <p>
-     * See https://kubernetes.io/docs/tasks/run-application/configure-pdb/#arbitrary-controllers-and-selectors for more
-     * details.
-     *
-     * @param name              Name of the PDB
-     * @param namespace         Namespace of the PDB
-     * @param labels            Labels of the PDB
-     * @param ownerReference    OwnerReference of the PDB
-     * @param template          PDB template with user's custom configuration
-     * @param replicas          Number of replicas in the resource(s) which this PDB covers
-     *
-     * @return The generated v1beta1 PodDisruptionBudget
-     */
-    public static io.fabric8.kubernetes.api.model.policy.v1beta1.PodDisruptionBudget createCustomControllerPodDisruptionBudgetV1Beta1(
-            String name,
-            String namespace,
-            Labels labels,
-            OwnerReference ownerReference,
-            PodDisruptionBudgetTemplate template,
-            int replicas
-    )   {
-        return new io.fabric8.kubernetes.api.model.policy.v1beta1.PodDisruptionBudgetBuilder()
-                .withNewMetadata()
-                    .withName(name)
-                    .withLabels(labels.withAdditionalLabels(TemplateUtils.labels(template)).toMap())
-                    .withNamespace(namespace)
-                    .withAnnotations(TemplateUtils.annotations(template))
-                    .withOwnerReferences(ownerReference)
-                .endMetadata()
-                .withNewSpec()
-                    .withMinAvailable(new IntOrString(Math.max(0, replicas - (template != null ? template.getMaxUnavailable() : DEFAULT_MAX_UNAVAILABLE))))
-                    .withSelector(new LabelSelectorBuilder().withMatchLabels(labels.strimziSelectorLabels().toMap()).build())
-                .endSpec()
-                .build();
-    }
+
```

Comment:
No need for the double empty line -> you probably don't need any empty line at the end.